english/use-the-fork-luke/use-the-fork.md

```diff
@@ -0,0 +1,35 @@
+The hand of marshmallow has left you with so many questions. What did it want with the lighter? What does it want with me? Why did it have a ring-finger tan line? 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.